shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/config/impl/StorageContainerConfigurationFactory.java

```diff
@@ -15,43 +15,36 @@
  * limitations under the License.
  */
 
-package org.apache.shardingsphere.test.integration.container.config;
+package org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.impl;
 
 import lombok.AccessLevel;
 import lombok.NoArgsConstructor;
 import org.apache.shardingsphere.infra.database.type.DatabaseType;
 import org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.StorageContainerConfiguration;
-import org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.impl.mysql.DefaultMySQLContainerConfiguration;
-import org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.impl.opengauss.DefaultOpenGaussContainerConfiguration;
-import org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.impl.postgresql.DefaultPostgreSQLContainerConfiguration;
+import org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.impl.mysql.MySQLContainerConfigurationFactory;
+import org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.impl.opengauss.OpenGaussContainerConfigurationFactory;
+import org.apache.shardingsphere.test.integration.env.container.atomic.storage.config.impl.postgresql.PostgreSQLContainerConfigurationFactory;
 
 /**
- * Suite storage container configuration factory.
+ * Storage container configuration factory.
  */
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
-public final class SuiteStorageContainerConfigurationFactory {
+public final class StorageContainerConfigurationFactory {
     
     /**
      * Create new instance of storage container configuration.
-     *
+     * 
      * @param databaseType database type
-     * @param scenario scenario
      * @return created instance
      */
-    public static StorageContainerConfiguration newInstance(final DatabaseType databaseType, final String scenario) {
+    public static StorageContainerConfiguration newInstance(final DatabaseType databaseType) {
         switch (databaseType.getType()) {
             case "MySQL":
-                if ("default".equalsIgnoreCase(scenario)) {
-                    return new DefaultMySQLContainerConfiguration();
-                }
-                return new DefaultMySQLContainerConfiguration();
+                return MySQLContainerConfigurationFactory.newInstance();
             case "PostgreSQL":
-                if ("default".equalsIgnoreCase(scenario)) {
-                    return new DefaultPostgreSQLContainerConfiguration();
-                }
-                return new DefaultPostgreSQLContainerConfiguration();
+                return PostgreSQLContainerConfigurationFactory.newInstance();
             case "openGauss":
-                return new DefaultOpenGaussContainerConfiguration();
+                return OpenGaussContainerConfigurationFactory.newInstance();
             case "H2":
                 return null;
```

Comment:
Why is null here?

Reply:
that's because H2 container doesn't need any configuration. do you have any better idea ? I don't think `return null;` is a good idea, but there is no other better logic to make this clear.